packages/font-glyphs/src/symbol/geometric/rotational.ptl

```diff
@@ -0,0 +1,275 @@
+				BallCenter
+
+		# Florette
+		create-glyph [MangleName 'blackAndWhiteFlorette6'] [MangleUnicode 0x273E] : glyph-proc
+			set-width Geom.Width
+			include : union
+				union.apply null : for-rotational-pointing 3 0.5 Geom.Size 0 : Petal normalSw (Geom.Size * 4.5 / 7)
+				union.apply null : for-rotational-pointing 3 0 Geom.Size 0 : WhitePetal normalSw (Geom.Size * 4.5 / 7)
+				BallCenter
+		create-glyph [MangleName 'blackFlorette5'] [MangleUnicode 0x273F] : glyph-proc
+			set-width Geom.Width
+			include : difference
+				union.apply null : for-rotational-pointing 5 0 Geom.Size 0 : Petal normalSw (Geom.Size * 5 / 7)
+				refer-glyph : MangleName 'blackSlightlySmallCircle'
+		create-glyph [MangleName 'whiteFlorette5'] [MangleUnicode 0x2740] : glyph-proc
+			set-width Geom.Width
+			include : union
+				union.apply null : for-rotational-pointing 5 0 Geom.Size 0 : WhitePetal normalSw (Geom.Size * 5 / 7)
+				refer-glyph : MangleName 'blackSlightlySmallCircle'
+		create-glyph [MangleName 'outlinedFlorette8'] [MangleUnicode 0x2741] : glyph-proc
+			set-width Geom.Width
+			local innerSw : clamp (normalSw / 2) normalSw (Geom.Size * 4 / 7 - normalSw * 2)
+			local offset : (Geom.Size * 4 / 7 - innerSw) / 2
+			include : union
+				difference
+					union
+						union.apply null : for-rotational-pointing 8 0 Geom.Size 0 : WhitePetal normalSw (Geom.Size * 4 / 7)
+						union.apply null : for-rotational-pointing 8 0 (Geom.Size - offset) 0 : Petal (innerSw / 2) innerSw
+					refer-glyph : MangleName 'blackSlightlySmallCircle'
+				refer-glyph : MangleName 'whiteSlightlySmallCircle'
+
+		# Balloon
+		create-glyph [MangleName 'balloonAsterisk4'] [MangleUnicode 0x2723] : glyph-proc
+			set-width Geom.Width
+			include : union.apply null : for-rotational-pointing 4 0 Geom.Size 0 : Balloon normalSw ballRad
+		create-glyph [MangleName 'balloonHeavyAsterisk4'] [MangleUnicode 0x2724] : glyph-proc
+			set-width Geom.Width
+			include : union.apply null : for-rotational-pointing 4 0 Geom.Size 0 : Balloon heavySw (ballRad * 1.35)
+		create-glyph [MangleName 'balloonAsterisk6'] [MangleUnicode 0x2749] : glyph-proc
+			set-width Geom.Width
+			include : union.apply null : for-rotational-pointing 6 0 Geom.Size 0 : Balloon normalSw ballRad
+
+		create-glyph [MangleName 'clubAsterisk4'] [MangleUnicode 0x2725] : glyph-proc
+			set-width Geom.Width
+			include : union.apply null : for-rotational-pointing 4 0 Geom.Size 0 : Club normalSw (ballRad * 1.35)
+
+		# Snowflake
+		create-glyph [MangleName 'snowflake'] [MangleUnicode 0x2744] : glyph-proc
+			set-width Geom.Width
+			include : union.apply null : for-rotational-pointing 6 0 Geom.Size 0 : Snowflake normalSw normalSw 6 0
+		create-glyph [MangleName 'snowflakeTight'] [MangleUnicode 0x2745] : glyph-proc
+			set-width Geom.Width
+			include : union.apply null : for-rotational-pointing 6 0 Geom.Size 0 : Snowflake normalSw normalSw 6 1
+		create-glyph [MangleName 'snowflakeHeavyChevron'] [MangleUnicode 0x2746] : glyph-proc
```

Comment:
`snowflakeHeavyChevron` is too thick per the preview image. Consider reduce the stroke width of both the center stems and the chevron, while keeping the chevron being thicker than the center stem.